lib/archethic/p2p/message.ex

```diff
@@ -1715,22 +1715,10 @@ defmodule Archethic.P2P.Message do
   end
 
   def process(
-        %NotifyLastTransactionAddress{
-          last_address: last_address,
-          genesis_address: genesis_address,
-          timestamp: timestamp
-        },
+        notify_last_tx_address_msg = %NotifyLastTransactionAddress{},
         _
       ) do
-    with {local_last_address, _} <-
-           TransactionChain.get_last_address(genesis_address),
-         true <- local_last_address != last_address do
-      TransactionChain.register_last_address(genesis_address, last_address, timestamp)
-
-      # Stop potential previous smart contract
-      Contracts.stop_contract(local_last_address)
-    end
-
+    process_notify_last_transaction_address(notify_last_tx_address_msg)
```

Comment:
I wanted to be able to unit test this implementation, and to keep the code simple

Reply:
I couldn't find test for how the message get processed, do we need to test this or it should be incorporated in a e2e test somewhere else ?

Reply:
Send message is mocked, but you can do something like Mocking the message, and in the mock just call `Message.process` and it will run the process function on the right message

Reply:
done